backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -409,7 +404,7 @@ export class ExperimentClientController {
     @Req()
     request: AppRequest,
     experiment: MarkExperimentValidator
-  ): Promise<MonitoredDecisionPoint> {
+  ): Promise<Omit<MonitoredDecisionPoint, 'createdAt' | 'updatedAt' | 'versionNumber'>> {
```

Comment:
Creating an explicit type will be safe just in case if we modify `MonitoredDecisionPoint`

Reply:
code updated